tests_e2e/tests/lib/cgroup_helpers.py

```diff
@@ -164,9 +164,14 @@ def check_log_message(message, after_timestamp=datetime.datetime.min):
     return False
 
 
-def get_unit_cgroup_paths(unit_name):
+def get_unit_cgroup_path(unit_name):
     """
-    Returns the cgroup paths for the given unit
+    Returns the cgroup path for the given unit. For v1, gets the path of the cpu controller.
     """
     cgroups_api = get_cgroup_api()
-    return cgroups_api.get_unit_cgroup_paths(unit_name)
+    unit_controlgroup_path = systemd.get_unit_property(unit_name, "ControlGroup")
+
+    if isinstance(cgroups_api, SystemdCgroupApiv1):
+        return os.path.join(cgroups_api.get_controller_mountpoints().get('cpu,cpuacct'), unit_controlgroup_path[1:])
```

Comment:
I think we can abstract this cgroup class?
why can't use cgroups_api. get_unit_cgroup() and get cgroup object and add method to get cgroup cpu or memory path based on function parameters or something like that?

Reply:
We could but it doesn't really make sense for v2 since all controllers are at the same path

Reply:
Ok, I think method we have here for constructing cgroup.procs path, at least we should change that